bubble-chat-scheduling/scripts/schedule.js

```diff
@@ -0,0 +1,112 @@
+var libraries = [
+  'https://cdnjs.cloudflare.com/ajax/libs/moment-timezone/0.5.13/moment-timezone.min.js',
+  'https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.17.0/moment.min.js',
+  'https://cdnjs.cloudflare.com/ajax/libs/moment-timezone/0.5.13/moment-timezone-with-data-2012-2022.min.js',
+  'https://cdnjs.cloudflare.com/ajax/libs/jstimezonedetect/1.0.4/jstz.js',
+  'https://cdnjs.cloudflare.com/ajax/libs/jquery/1.9.0/jquery.min.js'
+]
+
+
+function initialize() {
```

Comment:
It will be a good idea to add clear comments (in jsDoc format) for each function so it is readable.